ORStools/gui/directions_gui.py

```diff
@@ -135,14 +103,50 @@ def get_parameters(self):
         if self.dlg.avoidpolygon_group.isChecked():
             layer = self.dlg.avoidpolygon_dropdown.currentLayer()
             if layer:
-                polygons = _get_avoid_polygons(layer)
+                polygons = self._get_avoid_polygons(layer)
                 self.options['avoid_polygons'] = polygons
```

Comment:
Maybe we should load via `json.loads(polygons.asJson())` here? I'm unsure whether this behaves correctly if `polygons` is `None` though…